src/features/crops/components/Market.tsx

```diff
@@ -94,7 +94,11 @@ export const Market: React.FC = () => {
           onClick={() => handleMarketClick()}
         />
       )}
-      <Modal centered show={isOpen} onHide={() => setIsOpen(false)}>
+      <Modal
+        centered
+        show={isOpen}
+        onHide={() => (tourIsOpen ? null : setIsOpen(false))}
```

Comment:
Consider moving this above so the `onHide` function is undefined. This might help with some of the internal workings in React Bootstrap to know that there is no `onHide` function.

I.e. `onHide={tourIsOpen ? () =>setIsOpen(false) : undefined`